packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -6,7 +6,7 @@ const SizePlugin = require('size-plugin');
 const autoprefixer = require('autoprefixer');
 const requireRelative = require('require-relative');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const FixStyleOnlyEntriesPlugin = require('webpack-fix-style-only-entries');
```

Comment:
Bah. Will the fix still let this strip out the Webpack modules that have nothing in them? They should have no originalModule since they're not from an ESM source?

Reply:
should fix it, just commented it out for now, because it broke the tests